### ISSUE TITLE
Nightly build crashes: libzstd.1.dylib not found

### ISSUE THREAD

**Issue by atl285** [SoftFever/OrcaSlicer#3099]
### OrcaSlicer Version

1.9.0-dev

### OS version

macOS 13.6.1

### Additional system information

x86_64

### Printer

Bambu Lab P1P

### How to reproduce

1. simply start OrcaSlicer

### Actual results

Application crashes and in crashlog following is shown :

~~~
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               OrcaSlicer [7678]
Path:                  /Users/USER/Downloads/OrcaSlicer.app/Contents/MacOS/OrcaSlicer
Identifier:            com.softfever3d.orca-slicer
Version:               1.9.0-dev (???)
Code Type:             X86-64 (Native)
Parent Process:        launchd [1]
User ID:               502

Date/Time:             2023-12-12 23:05:58.1472 +0100
OS Version:            macOS 13.6.1 (22G313)
Report Version:        12
Anonymous UUID:        FC9320B2-6788-13E3-A304-8F3EEFB010DC

Sleep/Wake UUID:       E9AA9705-4481-45D4-ADBB-CD2C9B04E3F3

Time Awake Since Boot: 30000 seconds
Time Since Wake:       557 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Termination Reason:    Namespace DYLD, Code 1 Library missing
Library not loaded: /usr/local/opt/zstd/lib/libzstd.1.dylib
Referenced from: <82844E9F-3ECA-344D-B47E-C629BAF84CFA> /Users/USER/Downloads/OrcaSlicer.app/Contents/MacOS/OrcaSlicer
Reason: tried: '/usr/local/opt/zstd/lib/libzstd.1.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/opt/zstd/lib/libzstd.1.dylib' (no such file), '/usr/local/opt/zstd/lib/libzstd.1.dylib' (no such file), '/usr/lib/libzstd.1.dylib' (no such file, not in dyld cache)
(terminated at launch; ignore backtrace)

Thread 0 Crashed:
0   dyld                          	    0x7ff813526c52 __abort_with_payload + 10
1   dyld                          	    0x7ff813540fd7 abort_with_payload_wrapper_internal + 82
2   dyld                          	    0x7ff813541009 abort_with_payload + 9
3   dyld                          	    0x7ff8134c58f0 dyld4::halt(char const*) + 375
4   dyld                          	    0x7ff8134c2b71 dyld4::prepare(dyld4::APIs&, dyld3::MachOAnalyzer const*) + 4526
5   dyld                          	    0x7ff8134c13bd start + 1805


Thread 0 crashed with X86 Thread State (64-bit):
  rax: 0x0000000002000209  rbx: 0x0000000000000000  rcx: 0x00007ff7b0b80488  rdx: 0x00007ff7b0b808f0
  rdi: 0x0000000000000006  rsi: 0x0000000000000001  rbp: 0x00007ff7b0b804d0  rsp: 0x00007ff7b0b80488
   r8: 0x00007ff7b0b804f0   r9: 0x0000000000000000  r10: 0x000000000000007a  r11: 0x0000000000000246
  r12: 0x000000000000007a  r13: 0x00007ff7b0b808f0  r14: 0x0000000000000001  r15: 0x0000000000000006
  rip: 0x00007ff813526c52  rfl: 0x0000000000000246  cr2: 0x0000000000000000
  
Logical CPU:     0
Error Code:      0x02000209 
Trap Number:     133


Binary Images:
       0x10f37e000 -        0x113595fff com.softfever3d.orca-slicer (1.9.0-dev) <82844e9f-3eca-344d-b47e-c629baf84cfa> /Users/USER/Downloads/OrcaSlicer.app/Contents/MacOS/OrcaSlicer
    0x7ff8134bb000 -     0x7ff8135535ef dyld (*) <3df96f32-b9c9-3566-a6b7-4daebc6d6563> /usr/lib/dyld
~~~

### Expected results

OrcaSlicer is starting as normal.

### Project file & Debug log uploads

[OrcaSlicer-2023-12-12-230558.ips.zip](https://github.com/SoftFever/OrcaSlicer/files/13653701/OrcaSlicer-2023-12-12-230558.ips.zip)

### Checklist of files to include

- [X] Log file
- [x] Project file

**Comment by atl285** [comment]
Seems to be the same as in [issue #243](https://github.com/SoftFever/OrcaSlicer/issues/243)

**Comment by atl285** [comment]
Solved in 1.9.0
